### ISSUE TITLE
Help with the overfitting 

### ISSUE THREAD

**Issue by RudraniChangal** [RudraniChangal/NLP-Chatbot-using-GRU-RNN#1]
with every epoch, the training loss is getting reduced but at the same time testing loss is increasing, a classic example of overfitting .
